include/ibamr/IBFEInstrumentPanel.h

```diff
@@ -100,15 +99,25 @@ class IBFEInstrumentPanel
                             double data_time);
 
     /*!
-     * \brief output exodus files and .dat file for meshes and nodes.
+     * \return get the instrument dump interval.
      */
-    void outputMeterMeshes(IBAMR::IBFEMethod* ib_method_ops, int timestep_num, double data_time);
-
+    int getInstrumentDumpInterval() const;
+    
     /*!
-     * \return return the instrument dump interval.
+     * \return get a pointer to the jjth meter mesh
      */
-    int getInstrumentDumpInterval() const;
-
+    libMesh::SerialMesh* getMeterMesh(const unsigned int jj) const;
+    
+    /*!
+     * \return get a pointer to the EquationSystem object for jjth meter mesh
+     */
+    libMesh::EquationSystems* getMeterMeshSystem(const unsigned int jj) const;
```

Comment:
As above, let's change the return type to `libMesh::EquationSystems&`.

Also, let's go ahead and change the method name to `getMeterMeshEquationSystems`, since `System` is a separate concept from `EquationSystems` in libMesh.